### PR TITLE
Add Member Participation Types

### PR DESCRIPTION
**Pull request by wordshaker**
## Description of Change
- I've written up some example profiles of those who might want to be a member of the .NET Foundation and why. 

## Extra Notes
- PLEASE review, make comments etc. I need feedback from people who identify with each of these groups to see what I have wrong/is missing
- Let me know if I've missed a group or piece of information
- If there is anyone arty who fancies drawing some pictures to go with these, it might make the post a little nicer to engage with.
- I'll give this another read through later as well, I've typed it out and ran so I imagine there will be some bits that need work

### CODE REVIEW

**Comment by BillWagner** [comment]
@wordshaker I like the direction you're taking here.

I'm about to post the minutes of our last meeting. After I do that, you'll need to rebase this and move those as well. Or, I can make the move as part of that PR, which might also mean you'll need to rebase.

Which will be easier?

**Comment by wordshaker** [comment]
I can rebase and move them, no problem at all 👍🏼

**Comment by wordshaker** [comment]
Hopefully, those changes work. Did we decide what to do about Dev Rels?

**Comment by wordshaker** [comment]
@BillWagner - I've had a thought about a previous discussion the committee had. Do these member profiles cover:

- technologists in .NET that aren't developers? Do we have a design or UX member base like JavaScript do, or are we not established enough in these communities?

- Game/Unity developers. Though I imagine this member base is covered by OSS, knowledge sharing and educators, are we missing any profile from this community?

I think we should go ahead with what we have for now, but that we may need to revisit the above at some point.

**Review by BillWagner — APPROVED** [review]
This looks great @wordshaker 

I had one small wording suggestion, based on our discussions this morning. LMK what you think. Then, I'll merge this and start promoting it with the marketing committee.

File: member-profiles.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# Member Participation Types
+
+The .NET Foundation welcomes participation and input from it's many types of member. The Foundation wants to benefit the whole .NET ecosystem and recognizes that there are many types of technologist within this.
+
+This document outlines some of the different types of .NET Foundation member. It includes:
```
[the remark this answers is not in the document]
Comment (wordshaker) [reply member-profiles.md:5]: I like this change. Good idea.

**Pull request merged**